conda.recipe/run_test.bat

```diff
@@ -1 +1 @@
-"%PREFIX%\Scripts\npm.cmd" install -g npm@latest --no-spin && "%PREFIX%\Scripts\npm.cmd" install . --no-spin --no-progress && "%PREFIX%\Scripts\npm.cmd" run test --no-progress --no-spin && if errorlevel 1 exit 1
+"%PREFIX%\npm.cmd" install . --no-spin --no-progress && "%PREFIX%\npm.cmd" run test --no-progress --no-spin && if errorlevel 1 exit 1
```

Comment:
Now that we have official builds of nodejs, we should not be monkeying with the npm that we get from there. This is left over from when we would sometimes have `0.10.x` as the best possible package on some platforms. Hopefully, this should be faster and more stable, and if something is broken, we have someone to complain to :)